### ISSUE TITLE
Counting number of visitors

### ISSUE THREAD

**Issue by OmkarAcharekar** [MLH-Fellowship/pod-22.P3.1-portfolio#46]
I thought this idea of counting no of visitors on our website and displaying it.. That would be interesting... It would be hard considering the time we left..But would be a good addition to the website ... Would like more suggestions on this?

**Comment by abhishekrawe** [comment]
Yes, it's a Good Feature. something like the number of visitor counters, the number of developers contributed counter and changes reflected with the refresh.
